src/sentry/replays/endpoints/organization_replay_index.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.organization import NoProjects, OrganizationEndpoint
+from sentry.models.organization import Organization
+from sentry.replays.utils import proxy_replays_service
+
+
+class OrganizationReplayIndexEndpoint(OrganizationEndpoint):
+    private = True
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        if not features.has("organizations:session-replay", organization, actor=request.user):
+            return Response(status=404)
+
+        try:
+            filter_params = self.get_filter_params(request, organization, date_filter_optional=True)
```

Comment:
im not sure how the replay backend works but do you not require a date filter?

Reply:
ah good call, yeah we do.